man/man4/zfs.4

```diff
@@ -2265,6 +2265,14 @@ If
 .Sy 0 ,
 generate a system-dependent value close to 6 threads per taskq.
 .
+.It Sy zio_taskq_wr_iss_ncpus Ns = Ns Sy 32 Pq uint
```

Comment:
```suggestion
.It Sy zio_taskq_wr_iss_ncpus Ns = Ns Sy 0 Pq uint
```

And the language should be updated accordingly to indicate it's determine dynamically unless overridden.

Reply:
Fixed, thanks (sorry, I should have seen that).